### ISSUE TITLE
RFE Android: Much Simpler Interface for smaller screens.

### ISSUE THREAD

**Issue by rgleason** [rgleason/weather_routing_pi#122]
Using Weather_routing on Android is very very difficult and impractical as the PI stands now.
We should discuss here how to simplify the interface so that routing can be done on a cell phone easily.

There is one good examples for how this can be done, look at  SailGrib.
I expect that weatherrouting would have more parameters, plus weatherrouting allows setting up and running multiple configurations, so it is more complex.

But we should think about how this could be 
1. Simpler
2. Less prone to failures
3. Easier to get current gribs.
4. Better reporting 

All on smaller screens.

**Comment by rgleason** [comment]
Process
1. Grib_pi  Select area and download.
2. WR_pi Set start and end marks.
3. WR_pi Set intermediate port/starboard marks.
4. Set start time.
5. Hit "Compute"
All other settings can be preset beforehand: Polars, grib/climatology, constraints and Advanced Tab.

How to reduce the size of the menus?

**Comment by sebastien-rosset** [comment]
I agree. In similar software, capabilities are typically accessible through menus, not by displaying a window panel that consumes screen real estate.

As a partial improvement, #221 could help a bit by completely removing the list of positions.
